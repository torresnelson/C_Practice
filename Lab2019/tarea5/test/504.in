# prueba de heap: eliminar_menor
insertar_en_heap (8,twr)
menor
eliminar_menor
es_vacio_heap
max_valor
insertar_en_heap (8,twr)
menor
insertar_en_heap (4,ggg)
menor
eliminar_menor
menor
insertar_en_heap (9999,twr)
insertar_en_heap (148,vbh)
insertar_en_heap (5000,sder)
menor
eliminar_menor
menor
insertar_en_heap (2,kiu)
menor
insertar_en_heap (4,ggg)
insertar_en_heap (8,twr)
eliminar_menor
menor
eliminar_menor
eliminar_menor
eliminar_menor
eliminar_menor
menor
es_vacio_heap
eliminar_menor
es_vacio_heap
Fin
