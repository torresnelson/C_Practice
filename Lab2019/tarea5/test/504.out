1>#  prueba de heap: eliminar_menor.
2>Se insertó el elemento.
3>(8,twr)
4>Eliminado.
5>Vacio.
6>Máximo valor posible: 10000.
7>Se insertó el elemento.
8>(8,twr)
9>Se insertó el elemento.
10>(4,ggg)
11>Eliminado.
12>(8,twr)
13>Se insertó el elemento.
14>Se insertó el elemento.
15>Se insertó el elemento.
16>(8,twr)
17>Eliminado.
18>(148,vbh)
19>Se insertó el elemento.
20>(2,kiu)
21>Se insertó el elemento.
22>Se insertó el elemento.
23>Eliminado.
24>(4,ggg)
25>Eliminado.
26>Eliminado.
27>Eliminado.
28>Eliminado.
29>(9999,twr)
30>No vacío.
31>Eliminado.
32>Vacio.
33>Fin.
